pandas/core/internals.py

```diff
@@ -214,21 +214,21 @@ def make_block(self, values, placement=None, ndim=None, **kwargs):
         if ndim is None:
             ndim = self.ndim
 
-        return make_block(values, placement=placement, ndim=ndim, **kwargs)
+        return make_block(values, placement=placement, ndim=ndim)
 
-    def make_block_scalar(self, values, **kwargs):
+    def make_block_scalar(self, values):
         """
         Create a ScalarBlock
         """
         return ScalarBlock(values)
 
-    def make_block_same_class(self, values, placement=None, fastpath=True,
-                              **kwargs):
+    def make_block_same_class(self, values, placement=None, ndim=None):
         """ Wrap given values in a block of same type as self. """
         if placement is None:
             placement = self.mgr_locs
-        return make_block(values, placement=placement, klass=self.__class__,
-                          fastpath=fastpath, **kwargs)
+        # TODO: Why not set default for ndim like we do for self.make_block?
```

Comment:
I don't think we should introduce such todo questions

Reply:
agree

Reply:
Will remove. Any thoughts on the answer?